src/dotnet/APIView/CloneAPIViewDB/CloneAPIView/ReviewModelOld.cs

```diff
@@ -312,7 +331,14 @@ public class PullRequestModel
     public class MappingModel
     {
         [JsonProperty("id")]
+        public string ReviewOldId { get; set; }
```

Comment:
Are you going to use this in the script, or its for later use?

Reply:
I do use it in this script when migrating comments, samples etc as well as to support incremental time stamp check for each older review.